packages/vector_graphics_compiler/lib/src/svg/masking_optimizer.dart

```diff
@@ -0,0 +1,320 @@
+import 'dart:core';
```

Comment:
Dart core is implicitly imported everywhere, so you don't need the import

```suggestion
```

Reply:
fixed.